tracing/src/test/java/com/palantir/tracing/TracersTest.java

```diff
@@ -59,13 +59,13 @@ public void before() {
 
         Tracer.setSampler(AlwaysSampler.INSTANCE);
         // Initialize a new trace for each test
-        Tracer.setTrace(Trace.of(true, "defaultTraceId", Optional.empty()));
+        Tracer.initTraceWithSpan(Observability.SAMPLE, "defaultTraceId", "rootOperation", SpanType.LOCAL);
```

Comment:
Fixes the test, a sampled trace with zero spans is an illegal state that doesn't exist in the wild.